src/scripts/cfo.zig

```diff
@@ -240,6 +225,166 @@ fn run_fuzzers(
+            .{},
+        );
+
+        for (pr_list) |pr| {
+            for (pr.labels) |label| {
+                if (stdx.cut(label.name, "fuzz ") != null) break;
+            } else continue;
+
+            const pr_directory = try shell.print("./working/{d}", .{pr.number});
+
+            try shell.exec(
+                "git clone https://github.com/tigerbeetle/tigerbeetle {directory}",
+                .{ .directory = pr_directory },
+            );
+
+            try shell.pushd(pr_directory);
+            defer shell.popd();
+
+            try shell.exec(
+                "git fetch origin refs/pull/{pr_number}/head",
+                .{ .pr_number = pr.number },
+            );
+            try shell.exec("git switch --detach FETCH_HEAD", .{});
+
+            const commit = try run_fuzzers_commit_info(shell);
+            log.info("fuzzing commit={s} timestamp={d}", .{ commit.sha, commit.timestamp });
+
+            shell.zig("build -Drelease build_fuzz", .{}) catch |err| {
+                log.warn("failed to build: {}", .{err});
+            };
+
+            var pr_fuzzers_count: u32 = 0;
+            for (std.enums.values(Fuzzer)) |fuzzer| {
+                const labeled = for (pr.labels) |label| {
+                    if (stdx.cut(label.name, "fuzz ")) |cut| {
+                        if (std.mem.eql(u8, cut.suffix, @tagName(fuzzer))) {
+                            break true;
+                        }
+                    }
+                } else false;
+
+                if (labeled or fuzzer == .canary) {
+                    pr_fuzzers_count += 1;
+                    try working_directory.append(pr_directory);
+                    try seed_record.append(.{
+                        .commit_timestamp = commit.timestamp,
+                        .commit_sha = commit.sha,
+                        .fuzzer = fuzzer,
+                        .branch = try shell.print(
+                            "https://github.com/tigerbeetle/tigerbeetle/pull/{d}",
+                            .{pr.number},
+                        ),
+                    });
+                }
+            }
+            assert(pr_fuzzers_count >= 2); // The canary and at least one different fuzzer.
+        }
+    }
+    const task_pr_count: u32 = @intCast(seed_record.items.len - task_main_count);
+
+    // Split time 50:50 between fuzzing main and fuzzing labeled PRs.
+    const weight = try shell.arena.allocator().alloc(u32, working_directory.items.len);
+    var weight_main_total: usize = 0;
+    var weight_pr_total: usize = 0;
+    for (weight[0..task_main_count]) |*weight_main| {
+        weight_main.* = @max(task_pr_count, 1);
+        weight_main_total += weight_main.*;
+    }
+    for (weight[task_main_count..]) |*weight_pr| {
+        weight_pr.* = @max(task_main_count, 1);
+        weight_pr_total += weight_pr.*;
+    }
+    if (weight_main_total > 0 and weight_pr_total > 0) {
+        assert(weight_main_total == weight_pr_total);
+    }
```

Comment:
Out of scope for this PR, but (within `main`) would it also make sense to weight different fuzzers differently? e.g. `SegmentedArray` is quite static, whereas `Forest` covers a lot of tricky stuff.